docs/firmware/golioth-firmware-sdk/firmware-upgrade/README.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Over-the-Air (OTA) Upgrade Overview
+sidebar_position: 1
+slug: firmware-upgrade
+---
+
+This section will guide you through Over-The-Air (OTA) firmware updates for
+embedded devices using the Golioth Device Firmware Update (DFU) service. The
+Golioth Firmware SDK includes DFU sample code which runs on all Continuously
+Verified Boards. A walkthrough is included here to help you become familiar with
+the process.
+
+## OTA Overview
+
+### Device Side
+
+The firmware update process is based around the open source
+[MCUboot](https://www.mcuboot.com/) project. MCUboot uses multiple flash memory
```

Comment:
Is this true for all ports?

Reply:
Hmmm, probably not for ESP-IDF. Do you have any advice on how to handle this portion of the documentation in an abstract way?

Reply:
I don't know that we need to describe what happens on the device in much detail. The service we provide is for getting the right artifact from the cloud to the device. What the device does with that artifact (i.e. how the update is applied) is an exercise for the user (with reference implementations for our standard ports). I know that right now the update is largely under the control of the SDK, but we've talked about moving that out to the app like in the Zephyr SDK, and so I think it's ok for the docs to move in that direction and be a little bit more vague about what happens on the device.

Reply:
@sam-golioth that helps clarify, thanks!

I respun that paragraph, please let me know what you think.

Reply:
looks good to me!